src/server/server.js

```diff
@@ -25,6 +25,19 @@ app.get("/weather/:lat/:lng", (req, res) => {
   });
 });
 
+app.get("/stock/:nse", (nse) => {
```

Comment:
`app.get` takes in a url, and a function whose parameters are "response" and "request" or `res` and `req`. Not `nse`. That should also fix the error where `nse` is being defined twice (once by argument and again below this line).